pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -63,10 +65,17 @@
  */
 public class PulsarClusterMetadataSetup {
 
+    private static final int DEFAULT_BUNDLE_NUMBER = 16;
+
     private static class Arguments {
         @Parameter(names = { "-c", "--cluster" }, description = "Cluster name", required = true)
         private String cluster;
 
+        @Parameter(names = {"-bc", "--broker-conf"},
+                description = "Configuration file for Broker to get the default bundle numbers "
```

Comment:
Seems a bit confusing if we only use "DefaultNumberOfNamespaceBundles" from this config file. Most of other parameters in this `Arguments` appears in the config file too.

Reply:
In the previous implementation, the bundle number for the default namespaces `public/default` is hardcoded as 16. 
What we are interested in is providing an option to set the bundle number from the user side. Here we get from the `broker.conf`. So, I think it has nothing to do with other configurations. 
BTW, we can get other parameters from the `broker.conf`,  but considering the compatibility(for now, all parameters are from the command line, by default, it has higher priority than the config file).  So, I think we can keep the way it works now. WDYT?

Reply:
> In the previous implementation, the bundle number for the default namespaces `public/default` is hardcoded as 16.
> What we are interested in is providing an option to set the bundle number from the user side.

Make he bundle number for the default namespaces configurable make perfect sense to me.




> As you said, we can get other parameters from the `broker.conf`, but considering the compatibility(for now, all parameters are from the command line, by default, it has higher priority than the config file). I think we can keep the way it works now. WDYT?

I just wonder what's wrong with parameter like "--default-namespace-bundles"?

Reply:
> This could be confusing because if the namespace is already existing, it won't have any effect. Also it's not clear in the command line that this will be related to the public/default namespace.

> Another approach could be to remove the 16 bundles in the hardcoded from and instead just use the default number of bundles which is set in broker.conf

As @merlimat said, the default namespace may already exist(because the `initialize-cluster-metadata` command can be executed repeatedly), at this point, parameter passed by `--default-namespace-bundles` will not take effect